src/inspect_ai/model/_call_tools.py

```diff
@@ -36,6 +36,7 @@
 from inspect_ai._util.content import (
     Content,
     ContentAudio,
+    ContentData,
```

Comment:
We need to also add `ContentData` to **inspect_ai.model.qmd**

Reply:
Done